site/_collections/algolia.js

```diff
@@ -71,9 +71,11 @@ const algoliaCollection = collections => {
   });
 
   return toIndex.map(item => {
-    const image = item.data.hero
-      ? generateImgixSrc(item.data.hero, {w: 100, auto: 'format'})
-      : '';
+    const thumbnail = item.data.thumbnail ?? item.data.hero;
+    const image =
+      thumbnail !== null
+        ? generateImgixSrc(thumbnail, {w: 100, auto: 'format'})
+        : '';
```

Comment:
Sorry, nit: as GitHub doesn't charge per character, I think we should be explicit where we can.

```suggestion
    let image = item.data.thumbnail ?? item.data.hero ?? '';
    if (image) {
      image = generateImgixSrc(thumbnail, {w: 100, auto: 'format'})
    }
```

If that makes sense?